SearchActivity.py

```diff
@@ -79,7 +66,13 @@ def __init__(self, handle):
 
         self._game = Game(canvas, parent=self, path=self.path,
                           colors=self.colors)
-        self._setup_presence_service()
+        self.connect('shared', self._shared_cb)
+        self.connect('joined', self._joined_cb)
+
+        self._collab = CollabWrapper(self)
+        self._collab.connect('message', self._message_cb)
+        self._collab.connect('joined', self._joined_cb)
```

Comment:
It'll be better to have the joined signal connected from one API to one callback, not having one from two different APIs to same callback.